# should be number_input or some other valid function: @test err_import
from pycasio.casio.input import notafunction
